src/rating/rating.ts

```diff
@@ -100,6 +100,13 @@ export class NgbRating implements ControlValueAccessor, OnInit, OnChanges {
 	 */
 	@Input() resettable: boolean;
 
+	/**
+	 * Allows setting a custom function to provide aria text
+	 */
+	@Input() set ariaText(f: (rating: NgbRating) => string) {
```

Comment:
why not simply this? why setters and private values?

`@Input() ariaText: (current: number, max: number) => string;`

Reply:
It also should stay `ariaValueText` otherwise it's not clear which text... I would just modify existing function, as it is not technically a public API anyway.